app/main.py

```diff
@@ -49,6 +64,9 @@ def poll(self) -> None:
             if pool.get_pool_data():
                 pool.perform_topup()
 
+        return self.config["check_interval_minutes"] > 0
```

Comment:
I don't like this in here. It's not part of the polling logic.

Reply:
Refactor without your alternative?

Reply:
nvm -- 4ee4f80 addresses this